src/audit/routes/maintain.py

```diff
@@ -146,5 +147,34 @@ def validate_login_log(data):
     handle_timestamp(data)
 
 
+@router.post("/log/admin_action", status_code=HTTP_201_CREATED)
+async def create_admin_action_log(
+    body: CreateAdminActionLogInput,
+    background_tasks: BackgroundTasks,
+    auth=Depends(Auth),
+) -> None:
+    """
+    Create a new `admin_action` audit log.
+
+    This endpoint does not include any authorization checks, but it is not
+    exposed and is only meant for internal use.
+
+    If the timestamp is omitted from the request body, the current date and
+    time will be used.
+
+    The response is returned _before_ inserting the new audit log in the
+    database, so that POSTing audit logs does not impact the performance of
+    the caller and audit-service failures are not visible to users.
```

Comment:
I'm assuming these failures still result in something being written to log files...

Reply:
Yes, some log is saved here: https://github.com/uc-cdis/audit-service/pull/74/files#diff-68f190167c7b60af8a30854666b03a7cb78b7241db2c5856c115caf854cd2d67R175
```
logger.debug(f"Creating `admin_action` audit log. Received body: {data}")
```